mne/epochs.py

```diff
@@ -735,7 +735,7 @@ def _compute_mean_or_stderr(self, picks, mode='ave'):
         # pick channels
         if picks is None:
             picks = pick_types(evoked.info, meg=True, eeg=True, ref_meg=True,
-                               stim=False, eog=False, ecg=False,
+                               stim=False, eog=False, ecg=False, seeg=True,
                                emg=False, exclude=[])
```

Comment:
hum where is "auto" used.

Basically None is the default "smart" option. So for averaging you need the ref_meg as you'll need it for inverse modeling. But you don't need ref_meg for plotting or TFR or PSD ...

makes sense?